plugins/hls-splice-plugin/hls-splice-plugin.cabal

```diff
@@ -27,6 +27,7 @@ library
                   , dlist
                   , retrie
                   , shake
+                  , syb
```

Comment:
Using `syb` is going to be probably very slow without care.

Reply:
Yeah, but it is rather clumsy and fragile to GHC API change to pattern matching on entire `HsExpr`, `HsPat` and `HsType`, so some kind of use of `syb` should be reasonable.

Reply:
I tried not to traverse out-of-scope branches in https://github.com/haskell/haskell-language-server/pull/759/commits/f2efe7cc1d0a10e84eec00a684253559ad2bde44. Hopefully this should speed up the usage of syb (though there have been no noticable slow down introduced by syb so far)